debian/control

```diff
@@ -11,6 +11,7 @@ Build-Depends: debhelper (>= 10),
  libdbus-1-dev (>= 1.0),
  libdbus-glib-1-dev,
  maemo-system-services-dev,
+ libsystemd-dev,
```

Comment:
Is there a way to make this optional (will this somehow cause trouble in Devuan)?

Reply:
It doesn't add troubles to Devuan. I already tested this to work in ascii-devel.
Devuan has support for libsystemd-dev and many packages in Devuan are linked to libsystemd0.
The purpose of Devuan is to not use systemd as init system, but it still uses it's libraries.
Anyway, this is just a build-time dependency which doesn't affect the resulting package for Devuan in any way.

Reply:
Thanks for clarifying.

Reply:
FWIW, in Devuan we also have libraries that can replace libsystemd and keep API compatibility.